app/models/experience.server.ts

```diff
@@ -0,0 +1,21 @@
+
+import { prisma } from "../db.server";
+
+export async function createExperience(comentario: string, profileId: string){
+    const result = prisma.experience.create({
+        data: { comentario: comentario, profile: { connect: { id: profileId } } },
+    })
+    
+    return result
+}
+
+export async function getExperience() {
+    const result =  prisma.experience.findMany({
+        include: { profile: {select: {avatarUrl: true }}}
+    });
+    return result
```

Comment:
add  ;

Reply:
maybe you missed restart VSC to have the eslint installation complete

Reply:
yes, in general the changes can be made automatically for you if you install the prettier VSCode extension